.github/workflows/ci.yml

```diff
@@ -197,6 +197,28 @@ jobs:
     - name: Distcheck
       run: make distcheck DISTCHECK_CONFIGURE_FLAGS="--enable-werror"
 
+  build-freebsd-latest-clang:
```

Comment:
I would say this is a "freebsd-vm", and not "freebsd", and thus I would name this build `build-freebsd-vm-14-clang`.

Reply:
Likewise, `build-netbsd-vm-9.3-gcc` `build-openbsd-vm-7.4-gcc`

Reply:
I do not mind making that change, does this mean the GitHub hosted runners are all bare metal (the ubuntu builds) and not running "vm"s?

Reply:
No. My idea was to distinguish OSes running inside a "container" and OSes that are fully virtualized. Since what you are doing is a VM running inside another VM. And I was initially confused when there is a `runs-on: ubuntu` line there.

Reply:
As if all the other builds are done on real hardware … ;-) The important part IMHO for those tests is that the kernel side of things is the real deal™, which is satisfied if you don't just containerize, but run a VM (para-virtualized or full).